src/lv_interop.h

```diff
@@ -93,6 +93,20 @@ namespace grpc_labview
             return (T*)(rawBytes + byteOffset);
     #endif
         }
+
+        void* bytes(int byteOffset, int byteAlignment)
+        {
+#ifndef _PS_4
+            int reminder = 4 % byteAlignment;
+            if (reminder == 0)
+            {
+                return (void*)(rawBytes + byteOffset);
+            }
+            return (void*)(rawBytes + byteAlignment - reminder + byteOffset); // 8-byte aligned data
```

Comment:
We've got at least 4 copies of alignment calculation that are not all the same. The templatized `bytes` functions are simpler because they know arrays are always maximally aligned and start with a 4 byte size. If you are writing array specific code, you can continue this. The code doesn't need to do a modulo. It can look like the previous function. Then to avoid duplication the templatized functions can call the new function passing in `sizeof` as `alignmentRequirement`.

If you take it one step further, all 3 can call `AlignClusterOffset` and remove even more duplication.

Reply:
I have updated the bytes() template methods to have static assert when used with struct/class types which would require the explicit calculation of byteAlignment. All of them now use the AlignClusterOffset method to calculate byte alignment. 

There was one other usage of the bytes() template method which passed a struct in grpc_interop.cc. I have updated that method as well to pass in an explicit byteAlignment instead.